lib/src/command/global_activate.dart

```diff
@@ -89,9 +105,14 @@ class GlobalActivateCommand extends PubCommand {
 
         validateNoExtraArgs();
         return globals.activateHosted(package, constraint, executables,
+            features: features,
             overwriteBinStubs: overwrite);
 
       case "path":
+        if (features.isNotEmpty) {
+          usageException("--features may not be used with the path source.");
```

Comment:
Because we don't run a new `pub get` when we globally activate a path dependency, we just use its existing lockfile.